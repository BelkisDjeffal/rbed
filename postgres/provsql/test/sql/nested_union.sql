\set ECHO none
SET search_path TO provsql_test,provsql;

CREATE TABLE a(f int);
CREATE TABLE b(f int);
INSERT INTO a VALUES(42);
INSERT INTO b VALUES(42);

SELECT add_provenance('a');
SELECT add_provenance('b');

CREATE TABLE c as SELECT * FROM (SELECT * FROM a UNION SELECT * FROM b) t;

SELECT remove_provenance('c');

SELECT * FROM c;

DROP TABLE a;
DROP TABLE b;
DROP TABLE c;
