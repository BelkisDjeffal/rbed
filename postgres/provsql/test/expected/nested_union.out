\set ECHO none
 add_provenance 
----------------
 
(1 row)

 add_provenance 
----------------
 
(1 row)

 remove_provenance 
-------------------
 
(1 row)

 f  
----
 42
(1 row)

